Resources/Prototypes/ai_factions.yml

```
@@ -210,3 +210,23 @@
+  - PetsNT
+  - SimpleHostile
+  - SimpleNeutral
+  - Syndicate
+  - Xeno
+  - Zombie
+  - Revolutionary
+  - Changeling # goob edit - changelings
+  - Heretic # goob edit - heretics
+  - Blob # Goobstation - Blob
+  - Wizard # goob edit - wizard
+  - PirateFaction # goob - pirates
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)